### ISSUE TITLE
[REQUEST] add a option in openbullet settings

### ISSUE THREAD

**Issue by Teals53** [openbullet/openbullet#90]
for when i stop check then start its reset all infos in runner. option for close or open this

**Comment by meinname** [comment]
Sorry i/we didn't really understand this.
Did you mean this:
When you stop a runner and start it again, the stats get reset.
You want an option for not resetting (as long as you don't load a other config / wordlist)
 
Or smth else like that stats need to survive even a restart of openbullet?

**Comment by Teals53** [comment]
second one yes not this "Or smth else like that stats need to survive even a restart of openbullet?"

**Comment by openbullet** [comment]
> Second one yes
> Not this (second one)
Pick one...

**Comment by Teals53** [comment]
its is second one :fappa:

"Did you mean this:
When you stop a runner and start it again, the stats get reset.
You want an option for not resetting (as long as you don't load a other config / wordlist)
So the stop button is more like a pause button if the option is enabled."

**Comment by mastercho** [comment]
i still did not get it

**Comment by meinname** [comment]
short: 
i was right and mr. openbullet not :D

**Comment by openbullet** [comment]
TL;DR @meinname is a lucky one

**Comment by openbullet** [comment]
Anyways sorry I won't do this, if it was the other option I might have given it a look
